src/main/charts/bitbucket/templates/NOTES.txt

```diff
@@ -39,9 +39,15 @@ Get the {{ title .Chart.Name }} URL by running these commands in the same shell:
 {{- end }}
 {{- end }}
 
+{{ if not ( or .Values.volumes.localHome.persistentVolumeClaim.create .Values.volumes.localHome.customVolume ) }}
+#################################################################################
+######      WARNING: Persistent volume is not used for Local Home!!!       #####
```

Comment:
Minor indentation issue here you need one more space so that the hashes `#` are vertically aligned :) This applies to the other `NOTES.txt`

Reply:
Good spot!! Fixed~